README.md

```diff
@@ -1,6 +1,161 @@
-# molecule-javascript
+# Molecule-JavaScript
 
-## Molecule
+## Quick Start
+
+There's a [template project](https://github.com/keith-cy/molecule-javascript-template) for quick start to use `molecule-javascript`.
+
+It shows the most general way of using `molecule-javascript` in Browsers and Node.js with real-world demos.
+
+## Start
+
+### Use in Node.js
+
+```js
+import Molecule from '@nervosnetwork/molecule-javascript'
+
+const normalizedSchema = {
+  name: 'Bytes',
+  type: 'fixvec',
+  item: {
+    type: 'byte',
+  },
+}
+const data = ['0x01', '0x02']
+const molecule = new Molecule(normalizedSchema)
+const serialized = molecule.serialize(data) // expect to be "0x020000000102"
+const parsed = molecule.deserialize('0x020000000102') // expect to be ["0x01", "0x02"]
+```
+
+### Use in Browsers
+
+```js
+const normalizedSchema = {
+  name: 'Bytes',
+  type: 'fixvec',
+  item: {
+    type: 'byte',
+  },
+}
+const data = ['0x01', '0x02']
+const molecule = new MoleculeJavaScript.Molecule(normalizedSchema)
```

Comment:
Where `MoleculeJavaScript ` is from ?

Reply:
> Where `MoleculeJavaScript ` is from ?

It's exposed by `webpack` bundled file, namely the `lib/index.js`

Reply:
I suggest we should add description about this usage.

Reply:
> I suggest we should add description about this usage.

The info of umd js package has been updated in https://github.com/Keith-CY/molecule-javascript/pull/65/commits/5027fbbe64c72f664427a88aeb4ddf87c2117d68

Reply:
I mean we should add the desription which includes how to generate this package and how to import into Browser. For me, I didn't know what the `MoleculeJavaScript` is .

I think [It's exposed by webpack bundled file, namely the lib/index.js] is OK.

Reply:
> I mean we should add the desription which includes how to generate this package and how to import into Browser. For me, I didn't know what the `MoleculeJavaScript` is .
> 
> I think "It's exposed by webpack bundled file, namely the lib/index.js" is OK.

For users, they should not care about the implementation detail. So we should not mention `webpack` in the docs. If we indicate the `webpack` here, we have to update it once we transfer to `parcel` or `rollup`.

Reply:
I mean users do not care about the implementation detail, and I just suggest they should know what `MoleculeJavaScript ` is. 

If you think `The package is exposed as globalThis.MoleculeJavaScript` is OK, I think it is fine.

Reply:
> I mean users do not care about the implementation detail, and I just suggest they should know what `MoleculeJavaScript ` is.
> 
> If you think `The package is exposed as globalThis.MoleculeJavaScript` is OK, I think it is fine.

Yep, I think `globalThis.MoleculeJavaScript` is enough, it's the standard statement of the universal module(for both browsers and node.js) in javascript.